helm/prometheus-agent/charts/prometheus-agent/templates/prometheus.yaml

```diff
@@ -59,6 +59,10 @@ spec:
         name: {{ $remoteWrite.name }}-remote-write-api
     remoteTimeout: 5m
     url: {{ $remoteWrite.url }}
+    queueConfig:
```

Comment:
can we make them configurable in the values?

Reply:
Can you take care? I'll have to go to the doctor soon